aspnetcore/razor-components/index.md

```diff
@@ -77,51 +46,63 @@ A *component* is a .NET class. The class can either be written directly, as a C#
+## JavaScript interop
 
-## Code sharing and .NET Standard
+For apps that require third-party JavaScript libraries and browser APIs, Razor Components interoperate with JavaScript. Razor Components are capable of using any library or API that JavaScript is able to use. C# code can call into JavaScript code, and JavaScript code can call into C# code. For more information, see [JavaScript interop](xref:razor-components/javascript-interop).
 
-Apps can reference and use existing [.NET Standard](/dotnet/standard/net-standard) libraries. .NET Standard is a formal specification of .NET APIs that are common across .NET implementations. .NET Standard 2.0 or higher is supported. APIs that aren't applicable inside a web browser (for example, accessing the file system, opening a socket, threading, and other features) throw [PlatformNotSupportedException](/dotnet/api/system.platformnotsupportedexception). .NET Standard class libraries can be shared across server code and in browser-based apps.
+## Hosting models
 
-## JavaScript interop
+### Server-side hosting model
+
+Because Razor Components decouple a component's rendering logic from how UI updates are applied, there's flexibility in how Razor Components can be hosted. ASP.NET Core Razor Components in .NET Core 3.0 adds support for hosting Razor Components on the server in an ASP.NET Core app. UI updates are handled over a SignalR connection.
+
+The runtime:
 
-For apps that require third-party JavaScript libraries and browser APIs, WebAssembly is designed to interoperate with JavaScript. Razor Components are capable of using any library or API that JavaScript is able to use. C# code can call into JavaScript code, and JavaScript code can call into C# code. For more information, see [JavaScript interop](xref:razor-components/javascript-interop).
+* Handles sending UI events from the browser to the server.
+* Applies UI updates sent by the server back to the browser after running the components.
 
-## Optimization
+The connection used by Razor Components to communicate with the browser is also used to handle JavaScript interop calls.
 
-For client-side apps, payload size is critical. Blazor optimizes payload size to reduce download times. For example, unused parts of .NET assemblies are removed during the build process, HTTP responses are compressed, and the .NET runtime and assemblies are cached in the browser.
+![Razor Components runs .NET code on the server and interacts with the Document Object Model on the client over a SignalR connection](index/_static/aspnet-core-razor-components.png)
+
+For more information, see <xref:razor-components/hosting-models#server-side-hosting-model>.
 
-Razor Components provides an even smaller payload size than Blazor by maintaining .NET assemblies, the app's assembly, and the runtime server-side. Razor Components apps only serve markup, script, and stylesheets to clients.
+### Client-side hosting model
```

Comment:
The description of Blazor in this section is great, but I think it would be better to move most of this text to the new Blazor topic (#10825). Here we just need a brief paragraph here saying that Blazor let's you host Razor Components in the browser and then point off to the Blazor topic to learn more.

Reply:
I'll note this remark on #10825. When I work that ... *very soon* ... we can drop this in favor of the paragraph + link. I don't want to try to work #10825 concurrently on this PR, but I also don't want to leave a significant gap in coverage without a link to *something* that explains this out.